CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+### v0.8.1
+
+rkt v0.8.1 is an incremental release with numerous bug fixes and clean-up to the build system. It also introduces a few small new features and UX improvements.
+
+- New features and UX changes:
+  - `rkt rm` is now variadic: it can now remove multiple pods in one command, by UUID
+  - The `APPNAME` column in `rkt image list` output has been changed to the more accurate `NAME`. This involves a schema change in rkt's on-disk datastore, but this should be upgraded transparently.
+  - Headers are now sent when following HTTP redirects while trying to retrieve an image
+  - The default metadata service port number was changed from a registered/reserved IANA port to a random port in the non-dynamic range
```

Comment:
The default port is a random one? It will change randomly? Or we chose a random but fixed one as default?